src/cargo/ops/cargo_package.rs

```diff
@@ -81,7 +78,7 @@ pub fn package(ws: &Workspace<'_>, opts: &PackageOpts<'_>) -> CargoResult<Option
     if !pkg.manifest().exclude().is_empty() && !pkg.manifest().include().is_empty() {
         config.shell().warn(
             "both package.include and package.exclude are specified; \
-             the exclude list will be ignored",
+            the exclude list will be ignored",
```

Comment:
There seems to be several unrelated whitespace changes, can you try to avoid the extraneous changes?